src/components/NoClubsFound.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Button, Card, Heading, Image, Link } from "@hackclub/design-system";
+
+const Base = Card.extend.attrs({
+    bg: "snow",
+    boxShadowSize: "lg",
+    p: 4
+})`
+    align-items: center;
+    flex-direction: column;
+    max-width: 48rem;
+`;
+
+const SuperButton = Button.withComponent(Link).extend`
+    background-color: ${props => props.theme.colors.fuschia[6]};
+    background-image: linear-gradient(
+        -32deg,
+        ${props => props.theme.colors.fuschia[5]},
+        ${props => props.theme.colors.red[5]},
+        ${props => props.theme.colors.red[6]}
+    );
+`;
+
+// TODO: Find a better name
+const NoClubsFound = props => (
+    <Base>
```

Comment:
without seeing the site, this might look better centered?

Reply:
Done by its parent container here: https://github.com/hackclub/finder/blob/react-ds-rewrite/src/pages/index.js#L123